backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1200,6 +1189,11 @@ databaseChangeLog:
         - dropNotNullConstraint:
             tableName: api_user
             columnName: last_seen
+      rollback:
```

Comment:
I had the same `NOT NULL` problem as earlier here, but setting a default value of `NOW()` seems like a good solution.